### PR TITLE
Comment on a PR when screenshot tests fail

### PR DESCRIPTION
**Pull request by zgeor**
This PR enables commenting on a PR if the screenshot tests fail. 
One comment per failing flavour will be posted, specifying the paths to the screenshots that have failed. It does not add protection agains having all screenshots failing. 

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt [hunk 1]
```diff
@@ -127,7 +127,7 @@ open class BpkBadge @JvmOverloads constructor(
   private fun setup() {
     this.includeFontPadding = true
     this.textStyle = XS
-    this.weight = Weight.EMPHASIZED
+    this.weight = Weight.NORMAL
```
Comment (frugoman) [inline Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt:130]: is this change intended on this PR?

**Review by frugoman — CHANGES_REQUESTED** [review]

**Review by marianeum — APPROVED** [review]
🎉
